packages/frontity-org-theme/src/processors/terminal.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { css, styled } from "frontity";
+import { Processor } from "@frontity/html2react/types";
+import FrontityOrg from "../../types";
+
+const Dot = styled.span`
+  display: inline-block;
+  height: 8px;
+  width: 8px;
+  border-radius: 50%;
+  margin-left: 7px;
+  margin-top: 8px;
+  background-color: rgba(255, 255, 255, 0.15);
+`;
+
+const Top = () => (
+  <div
+    css={css`
+      height: 26px;
+      border-bottom: 1px solid rgba(255, 255, 255, 0.07);
+    `}
+  >
+    <Dot
+      css={css`
+        margin-left: 10px;
+      `}
+    />
+    <Dot />
+    <Dot />
+  </div>
+);
+
+const backgroundColor: Processor<React.HTMLProps<HTMLElement>, FrontityOrg> = {
+  name: "backgroundColor",
+  test: ({ node }) =>
+    node.type === "element" &&
+    (node.props?.className?.split(" ").includes("terminal") ||
+      node.props?.className?.split(" ").includes("wp-block-code") ||
+      node.component === "code"),
+
+  processor: ({ node, state }) => {
+    if (node.type !== "element") {
+      return node;
+    }
+
+    if (node.component === "code") {
+      node.props.css = css`
+        position: absolute;
+        margin-left: 15px;
+        margin-top: 10px;
+      `;
+      node.props.className = "language-javascript";
+      return node;
+    }
+
+    node.props.css = css`
+      ${node.props.css}
+      font-family: Consolas, Monaco, "Andale Mono", "Ubuntu Mono", monospace;
+      font-size: 0.78rem;
+      line-height: 1.65;
+      background: ${state.theme.colors.voidblu};
+      height: 310px;
```

Comment:
With that height, it would be necessary to add `overflow: auto;` here in case the code was higher than the terminal view. You can see what happen now in the preview at "Terminal long list": https://frontity-org-qgwypgj91.now.sh/common-styles/